CI is failing

Most recent failure

https://github.com/google/closure-compiler-npm/actions/runs/2384282215

The error in the CI output is:

```
Analyzing: target //:compiler_unshaded_deploy.jar (19 packages loaded, 6 targets configured)
ERROR: While resolving toolchains for target //:compiler_unshaded: invalid registered toolchain '//toolchains:all': while parsing '//toolchains:all': no such package 'toolchains': BUILD file not found in any of the following directories. Add a BUILD file to a directory to mark it as a package.
 - /home/runner/work/closure-compiler-npm/closure-compiler-npm/compiler/toolchains
ERROR: Analysis of target '//:compiler_unshaded_deploy.jar' failed; build aborted: invalid registered toolchain '//toolchains:all': while parsing '//toolchains:all': no such package 'toolchains': BUILD file not found in any of the following directories. Add a BUILD file to a directory to mark it as a package.
 - /home/runner/work/closure-compiler-npm/closure-compiler-npm/compiler/toolchains
INFO: Elapsed time: [19](https://github.com/google/closure-compiler-npm/runs/6536803773?check_suite_focus=true#step:10:20).758s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (19 packages loaded, 6 targets configured)
FAILED: Build did NOT complete successfully (19 packages loaded, 6 targets configured)
/home/runner/work/closure-compiler-npm/closure-compiler-npm/build-scripts/run-command.js:67
        const err = new Error(`non-zero exit code ${exitCode}`);
                    ^

Error: non-zero exit code 1
    at ChildProcess.<anonymous> (/home/runner/work/closure-compiler-npm/closure-compiler-npm/build-scripts/run-command.js:67:[21](https://github.com/google/closure-compiler-npm/runs/6536803773?check_suite_focus=true#step:10:22))
    at ChildProcess.emit (node:events:5[27](https://github.com/google/closure-compiler-npm/runs/6536803773?check_suite_focus=true#step:10:28):[28](https://github.com/google/closure-compiler-npm/runs/6536803773?check_suite_focus=true#step:10:29))
    at maybeClose (node:internal/child_process:1092:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:[30](https://github.com/google/closure-compiler-npm/runs/6536803773?check_suite_focus=true#step:10:31)2:5) {
  stdout: '',
  stderr: '',
  exitCode: 1
}
Error: Process completed with exit code 1.
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -25,10 +25,8 @@ jobs:
           java-version: 17
           java-package: jdk
           architecture: x64
-      - name: Setup Bazel
-        uses: jwlawson/actions-setup-bazel@v1
-        with:
-          bazel-version: '4.2.2'
+      - name: Setup Bazelisk
+        uses: bazelbuild/setup-bazelisk@v2
       - uses: actions/checkout@v2
         with:
           submodules: recursive

--- a/build-scripts/build_compiler.js
+++ b/build-scripts/build_compiler.js
@@ -44,7 +44,7 @@ async function main() {
   console.log(process.platform, process.arch, compilerVersion);
 
   const { exitCode } = await runCommand(
-    "bazel",
+    "bazelisk",
     [
       "build",
       "--color=yes",

